src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+                setDone(tasks, userCommand);
+            }
+            else {
+                int lineSize = userCommand.length();
+                showAddTaskMessage();
+                if (userCommand.startsWith("todo")) {
+                    taskCount = addTodo(tasks, taskCount, userCommand, lineSize);
+                }
+                else if (userCommand.startsWith("deadline")) {
+                    taskCount = addDeadline(tasks, taskCount, userCommand, lineSize);
+                }
+                else if (userCommand.startsWith("event")) {
+                    taskCount = addEvent(tasks, taskCount, userCommand, lineSize);
+                }
+                else {
+                    taskCount = addTask(tasks, taskCount, userCommand);
+                }
+                showTaskCountMessage(taskCount);
+            }
+        }
+    }
+
+    private static String getUserInput() {
+        Scanner input = new Scanner(System.in);
+        return input.nextLine();
+    }
+
+    private static int addTask(Task[] tasks, int taskCount, String userCommand) {
+        for (Task task : tasks) {
+            tasks[taskCount] = new Task(userCommand);
+        }
+        echoTask(tasks, taskCount);
+        taskCount++;
+        return taskCount;
+    }
+
+    private static int addTodo(Task[] tasks, int taskCount, String userCommand, int lineSize) {
+        String todoTask = getTask(userCommand);
+        for (Task task : tasks) {
+
+            tasks[taskCount] = new Todo(todoTask);
+        }
+        echoTask(tasks, taskCount);
+        taskCount++;
+        return taskCount;
+    }
+
+    private static int addDeadline(Task[] tasks, int taskCount, String userCommand, int lineSize) {
+        String deadlineTask = getTask(userCommand);
+        String deadlineBy = getDate(userCommand);
+        for (Task task : tasks) {
+            tasks[taskCount] = new Deadline(deadlineTask, deadlineBy);
+        }
+        echoTask(tasks, taskCount);
+        taskCount++;
+        return taskCount;
+    }
+
+    private static int addEvent(Task[] tasks, int taskCount, String userCommand, int lineSize) {
+        String eventTask = getTask(userCommand);
+        String eventAt = getDate(userCommand);
+        for (Task task : tasks) {
+            tasks[taskCount] = new Event(eventTask, eventAt);
+        }
+        echoTask(tasks, taskCount);
+        taskCount++;
+        return taskCount;
+    }
+
+    private static void setDone(Task[] tasks, String userCommand) {
+        int index = getDoneIndex(userCommand);
+        tasks[index].markAsDone();
+        showDoneMessage(tasks, index);
+    }
+
+    private static String getTask(String userCommand) {
+        if (userCommand.contains("/")) {
+            return userCommand.substring(userCommand.indexOf(" ") + 1, userCommand.indexOf("/") - 1);
+        }
+        else {
+            return userCommand.substring(userCommand.indexOf(" ") + 1);
+        }
```

Comment:
Should not have multiple  return lines in a single method

Reply:
You're right thanks for the feedback